src/main/java/org/mifos/identityaccountmapper/api/definition/FetchBeneficiariesApi.java

```diff
@@ -0,0 +1,21 @@
+package org.mifos.identityaccountmapper.api.definition;
+
+import org.mifos.identityaccountmapper.data.FetchBeneficiariesResponseDTO;
+import org.mifos.identityaccountmapper.data.ResponseDTO;
+import org.springframework.data.domain.Page;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestHeader;
+import org.springframework.web.bind.annotation.RequestParam;
+
+import java.util.concurrent.ExecutionException;
+
+public interface FetchBeneficiariesApi {
+    @GetMapping("/beneficiaries/{payeeIdentity}")
+    ResponseEntity<FetchBeneficiariesResponseDTO> fetchBeneficiary( @PathVariable String payeeIdentity,
+                                                                   @RequestHeader(value = "X-Registering-Institution-ID") String registeringInstitutionId) throws ExecutionException, InterruptedException;
+    @GetMapping("/beneficiaries")
+    ResponseEntity<Page<FetchBeneficiariesResponseDTO>> fetchAllBeneficiary(@RequestHeader(value = "X-Registering-Institution-ID") String registeringInstitutionId,  @RequestParam(value = "page", required = false, defaultValue = "0") Integer page,
```

Comment:
Please check the API guidelines for the pagination and filter params conventions

Reply:
As per my understanding this is per API guidelines, please suggest changes if any